pkgs/development/compilers/gcc/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, targetPackages, fetchurl, fetchpatch, noSysDirs
+{ lib, stdenv, targetPackages, fetchurl, fetchpatch, fetchFromGitHub, noSysDirs
```

Comment:
The failure to accept `fetchFromGithub` was masked by the global `with`! This is a real bug.